src/forms/StripePaymentForm/StripePaymentForm.js

```diff
@@ -306,6 +286,10 @@ StripePaymentForm.propTypes = {
   paymentInfo: string.isRequired,
   authorDisplayName: string.isRequired,
   showInitialMessageInput: bool,
+  onCreateStripePaymentToken: func,
```

Comment:
You should add `.isRequired` flag here or add default values in `StripePaymentForm.defaultProps`